src/ol_openedx_git_auto_export/README.md

```diff
@@ -0,0 +1,57 @@
+#### Steps to install on edX platform:
+
+
+You can install this plugin into any Open edX instance by using any of the following methods:
+
+
+**Option 1: Install from PyPI**
+
+.. code-block::
```

Comment:
This formatting is out of order check through markdown template on your branch. https://github.com/mitodl/open-edx-plugins/tree/umar/add-edx-github-export-plugin/src/ol_openedx_git_auto_export